packages/yew/src/dom_bundle/btag/attributes.rs

```diff
@@ -83,21 +84,26 @@ impl Apply for InputFields {
 
 impl Attributes {
     #[cold]
-    fn apply_diff_index_maps<'a, A, B>(
+    fn apply_diff_index_maps<'a>(
         el: &Element,
         // this makes it possible to diff `&'a IndexMap<_, A>` and `IndexMap<_, &'a A>`.
-        mut new_iter: impl Iterator<Item = (&'static str, &'a str)>,
-        new: &IndexMap<&'static str, A>,
-        old: &IndexMap<&'static str, B>,
-    ) where
-        A: AsRef<str>,
-        B: AsRef<str>,
-    {
+        mut new_iter: impl Iterator<Item = (&'a str, &'a str)>,
```

Comment:
A small gain (< 100bytes) is to change this to `&mut dyn Iterator<..>` instead of `impl`.